rj_geometry_msgs/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
```

Comment:
Weren't we going to delete `package.xml` files, considering we don't use them?

Reply:
Yes, except the `ros2` cli uses them, ie. if you don't have this for messages specifically then `ros2 topic` doesn't work.

Reply:
Sigh, okay.